server/repository/src/migrations/v1_07_00/changelog_add_name_link_id.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        DROP INDEX index_changelog_record_id;
+        DROP VIEW changelog_deduped;
+        "#
+    )?;
+
+    #[cfg(feature = "postgres")]
+    sql!(
+        connection,
+        r#"        
+        ALTER TABLE invoice DISABLE TRIGGER ALL;
+        ALTER TABLE invoice_line DISABLE TRIGGER ALL;
+        ALTER TABLE requisition DISABLE TRIGGER ALL;
+        ALTER TABLE requisition_line DISABLE TRIGGER ALL;
+        ALTER TABLE name_store_join DISABLE TRIGGER ALL;
```

Comment:
These are just the triggers which update the name_id, right? maybe add a comment?